windows/installer/MozillaVPN.wxs

```diff
@@ -11,85 +11,85 @@
+      <Directory Id="$(var.PlatformProgramFilesFolder)">
+        <Directory Id="MozillaFolder" Name="Mozilla">
+          <Directory Id="MozillaVPNFolder" Name="Mozilla VPN" />
+        </Directory>
+      </Directory>
+      <Directory Id="ProgramMenuFolder" />
+    </Directory>
+
+    <!--
 			Components
 		-->
-		<ComponentGroup Id="MozillaVPNComponents">
-			<Component Directory="MozillaVPNFolder" Id="MozillaVPNExecutable" Guid="EF353A73-A10B-4A2D-87AC-4389BBD9313B">
-				<File Id="MozillaVPNExecutable" Source="../../MozillaVPN.exe" KeyPath="yes">
-					<Shortcut Id="MozillaVPNShortcut" Directory="ProgramMenuFolder" Name="Mozilla VPN" Description="A fast, secure and easy to use VPN. Built by the makers of Firefox." WorkingDirectory="MozillaVPNFolder" Advertise="yes" />
-				</File>
-				<File Source="../../tunnel/$(var.Platform)/tunnel.dll" />
-				<File Source="../../tunnel/.deps/wintun/bin/$(var.WintunPlatform)/wintun.dll" />
-				<File Source="../../balrog/$(var.Platform)/balrog.dll" />
-				<File Source="../../libcrypto-1_1-x64.dll" />
-				<File Source="../../libssl-1_1-x64.dll" />
-				<File Source="../../libEGL.dll" />
-				<File Source="../../libGLESv2.dll" />
-				<File Source="../split-tunnel/mullvad-split-tunnel.cat" />
-				<File Source="../split-tunnel/mullvad-split-tunnel.inf" />
-				<File Source="../split-tunnel/mullvad-split-tunnel.sys" />
-				<File Source="../split-tunnel/WdfCoinstaller01011.dll" />
-				
-
-				<File Source="../../extension/manifests/windows/mozillavpn.json" />
-				<File Source="../../mozillavpnnp.exe" />
-
-				<!-- Broker Service -->
-				<ServiceInstall
+    <ComponentGroup Id="MozillaVPNComponents">
+      <Component Directory="MozillaVPNFolder" Id="MozillaVPNExecutable" Guid="EF353A73-A10B-4A2D-87AC-4389BBD9313B">
+        <File Id="MozillaVPNExecutable" Source="../../MozillaVPN.exe" KeyPath="yes">
+          <Shortcut Id="MozillaVPNShortcut" Directory="ProgramMenuFolder" Name="Mozilla VPN" Description="A fast, secure and easy to use VPN. Built by the makers of Firefox." WorkingDirectory="MozillaVPNFolder" Advertise="yes" />
+        </File>
+        <File Source="../../tunnel/$(var.Platform)/tunnel.dll" />
+        <File Source="../../tunnel/.deps/wintun/bin/$(var.WintunPlatform)/wintun.dll" />
+        <File Source="../../balrog/$(var.Platform)/balrog.dll" />
+        <File Source="../../libcrypto-1_1-x64.dll" />
+        <File Source="../../libssl-1_1-x64.dll" />
+        <File Source="../../libEGL.dll" />
+        <File Source="../../libGLESv2.dll" />
+        <File Source="../split-tunnel/mullvad-split-tunnel.cat" />
+        <File Source="../split-tunnel/mullvad-split-tunnel.inf" />
+        <File Source="../split-tunnel/mullvad-split-tunnel.sys" />
+        <File Source="../split-tunnel/WdfCoinstaller01011.dll" />
+
+
+        <File Source="../../extension/manifests/windows/mozillavpn.json" />
+        <File Source="../../extension/bridge/target/release/mozillavpnnp.exe" />
```

Comment:
Actually no... in theory the compilation script does:
```
                                                                                         
                                                                                                                 
IF %DEBUG_BUILD%==F (                                                                                            
  ECHO Generating Release Build for the extension bridge                                                         
  CD extension\bridge                                                                                            
                                                                                                                 
  cargo build --release                                                                                          
  IF %ERRORLEVEL% NEQ 0 (                                                                                        
    ECHO cargo failed for the extension!                                                                         
    EXIT 1                                                                                                       
  )                                                                                                              
                                                                                                                 
  CP target/release/mozillavpnnp.exe ..\..                                                                       
  CD ..\..                                                                                                       
)                                         
```

this should not be needed.

Reply:
It was failing on my builds.  Not sure what the difference is.  I just cloned a fresh repo...

Reply:
and the extension exe is not present in the root anymore after a build.